wordpress-mobile/gutenberg-mobile
# Release v1.62.2

Release for Gutenberg Mobile 1.62.2

This release contains the content of 1.62.1 plus the updated gutenberg release notes. 
It was created because buildkite wasn't able to overwrite the S3 artifact of 1.62.1 need for Android to include the change logs.

## Related PRs
Gutenberg: https://github.com/WordPress/gutenberg/pull/35225
WPAndroid: https://github.com/wordpress-mobile/WordPress-Android/pull/15391
WPiOS:       https://github.com/wordpress-mobile/WordPress-iOS/pull/17241

## To test:
Smoke test
See https://github.com/wordpress-mobile/gutenberg-mobile/pull/4031 

PR submission checklist:
